deploy/helm/sumologic/conf/traces/traces.otelcol.conf.yaml

```diff
@@ -1,4 +1,4 @@
 {{- $yamlFile := toYaml $.Values.otelcol.config }}
 {{- $fullname := include "sumologic.fullname" . }}
-{{- $url := list "http://" $fullname ":9411/api/v2/spans" | join "" }}
+{{- $url := list "http://" $fullname "-fluentd-logs:9411/api/v2/spans" | join "" }}
```

Comment:
So this appends `-fluentd-logs` to the url and the template is actually being used in both fluentd and otelcol config, right?

Reply:
We split metrics and logs in #588 so changed the service names as well.
We are using `-fluentd-logs`  and `-fluentd-metrics` suffixed services and tracing is going to use the logs one

Reply:
If we use fluentd for traces in the future it will probably get it's own fluentd.